plugins/entity-feedback-backend/src/service/router.ts

```diff
@@ -170,7 +181,16 @@ export async function createRouter(
       .filter(Boolean)
       .map(ent => stringifyEntityRef(ent!));
 
-    res.json(ratings.filter(r => accessibleEntityRefs.includes(r.userRef)));
+    res.json(
+      ratings.filter(
+        r =>
+          accessibleEntityRefs.includes(r.userRef) ||
+          user?.identity.userEntityRef === r.userRef ||
+          user?.identity.ownershipEntityRefs.includes(
+            entity?.spec!.ownerRef as string,
+          ),
```

Comment:
This can and should already be handled via an appropriate [permission policy](https://backstage.io/docs/permissions/writing-a-policy)